web/src/services/Client.ts

```diff
@@ -4,8 +4,8 @@ import { ServiceResponse, hasServiceError, toData } from "./Api";
 export async function PostWithOptionalResponse<T = undefined>(path: string, body?: any) {
     const res = await axios.post<ServiceResponse<T>>(path, body);
 
-    if (res.status !== 200 || hasServiceError(res)) {
-        throw new Error(`Failed POST to ${path}. Code: ${res.status}.`);
+    if (res.status !== 200 || hasServiceError(res)[0]) {
+        throw new Error(`Failed POST to ${path}. Code: ${res.status}. Message: ${hasServiceError(res)[1]}`);
```

Comment:
@clems4ever any thoughts as to if you're okay with handling bubbling the error up this way?
Is there another specific way you want this handled?

Reply:
I have a feeling this is not necessary to worry about. The only time this would theoretically be presented to an attacker is if they already compromised the users email right?